target/linux/ath79/image/generic.mk

```diff
@@ -676,6 +676,19 @@ define Device/librerouter_librerouter-v1
 endef
 TARGET_DEVICES += librerouter_librerouter-v1
 
+define Device/mikrotik_routerboard-wap-g-5hact2hnd
+  $(Device/mikrotik)
+  ATH_SOC := qca9556
```

Comment:
Please rename this to "SOC" due to 95030ac0b673165772daf27128359eaaf536d457.

Reply:
Fixed and rebased master. Thanks!